separate_instance/docker-compose.yml

```diff
@@ -0,0 +1,172 @@
+# This file defines the production settings. It is overwritten by docker-compose.override.yml,
+# which defines the development settings. The override.yml is loaded by default. Therefore it
+# is required to explicitly define if you want an production build:
+# > docker-compose -f docker-compose.yml up 
+
+version: "3.4"
+
+services:
+
+#  ########################################################
+#  # FRONTEND #############################################
+#  ########################################################
+#  frontend:
+#    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+#    image: gradido/frontend:local-production
+#    build:
+#      context: ./frontend
+#      target: production
+#    networks:
+#      - external-net
+#      - internal-net
+#    ports:
+#      - 3002:3002
+#    environment:
+#      # Envs used in Dockerfile
+#      # - DOCKER_WORKDIR="/app"
+#      # - PORT=3000
+#      # - BUILD_DATE="1970-01-01T00:00:00.00Z"
+#      # - BUILD_VERSION="0.0.0.0"
+#      # - BUILD_COMMIT="0000000"
+#      - NODE_ENV="production"
+#    # env_file:
+#      # - ./.env
+#      # - ./frontend/.env
+#
+#  ########################################################
+#  # ADMIN INTERFACE ######################################
+#  ########################################################
+#  admin:
+#    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+#    image: gradido/admin:local-production
+#    build:
+#      context: ./admin
+#      target: production
+#    networks:
+#      - external-net
+#      - internal-net
+#    ports:
+#      - 8081:8081
+#    environment:
+#      # Envs used in Dockerfile
+#      # - DOCKER_WORKDIR="/app"
+#      # - PORT=8090
+#      # - BUILD_DATE="1970-01-01T00:00:00.00Z"
+#      # - BUILD_VERSION="0.0.0.0"
+#      # - BUILD_COMMIT="0000000"
+#      - NODE_ENV="production"
+#    # env_file:
+#      # - ./.env
+#      # - ./admin/.env
+#
+  #########################################################
+  ## MARIADB ##############################################
+  #########################################################
+  mariadb:
+    build:
+      context: ./mariadb
+      target: mariadb_server
+    environment:
+      - MARIADB_ALLOW_EMPTY_PASSWORD=1
+      - MARIADB_USER=root
+      - MYSQL_TCP_PORT=3307
+    networks: 
+      - internal-net
+    ports: 
+      - 3307:3307
```

Comment:
if done properly you can override the port with an env variable:

`docker-compose up mariadb database backend --env DATABASE_PORT=3307`

Reply:
yes that's right, but was not the focus of this issue...
even that the previous port definition with 3306 was hardcoded too

Let us define a new issue for: 
- creating scripts to invoke different instances on different ports
- introducing variables in the existing config-, docker- and other-files, which were set outside from the scripts
- ...

all changes of this issue will be part of the root directory "separate_instance" in parallel to all other moduls and can be easiely deleted as soon as we had introduced a dynamic invocation per scripts and config-variables for the concerted modules.